.github/workflows/gen-upgrade-on-manual-input.yml

```diff
@@ -0,0 +1,60 @@
+# When user trigger this workflow with custom version input, this action will do 3 things:
+# 1) it will create a directory with an empty upgrade handler in app/upgrades folder
+# 2) will increase an E2E_UPGRADE_VERSION variable in Makefile
+# 3) create a pull request with these changes to main
```

Comment:
```suggestion
# 3) create a pull request with these changes to main

# Please note that if the given version (or release major version) is smaller than the latest
# existing upgrade handler version in `app/upgrades/`, this will be a no-op.
```